<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSignalPartitionDialog</class>
 <widget class="QDialog" name="EditSignalPartitionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Signal Partition</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>55</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Signal Partition</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_lsb">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LSB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>105</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxReg">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSigLSB">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSigMSB">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_msb">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MSB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>55</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>to</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Register Partition</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxRegPart">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBoxReg</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditSignalPartitionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditSignalPartitionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
